### PR TITLE
added GET members API

### LINKED ISSUE #36: Get Members API
Create an API for getting the members
URL: `GET /members`
The route should be accessed publicly. The response structure should be similar to the response schema for `GET /users.

URL: `GET /members/:userId`
Details to be added.

### PR DESCRIPTION
**Pull request by swarajpure**
Fixes #36 
- added GET `/members` API
- can be accessed publicly

Please review. Thank you!

### CODE REVIEW

File: models/members.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const userModel = firestore.collection('users')
+
+/**
+ * Fetches the data about our members
+ * @param query { Object }: Filter for members data
+ * @return {Promise<userModel|Array>}
+ */
+const fetchMembers = async (query) => {
+  try {
+    const snapshot = await userModel.get()
```
Comment (ankurnarkhede) [inline models/members.js:17]: @swarajpure , can you check if there is any way to directly query only those documents with `isMember` as `true`?

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

**Comment by whyDontI** [comment]
Please add contract for this API

File: models/members.js (hunk 1 above)
Comment (swarajpure) [reply models/members.js]: Done. Thank you!

**Comment by swarajpure** [comment]
> 
> 
> Please add contract for this API

Sure. Will do.

**Comment by Rucha1499** [comment]
Can you add the Swagger docs for this route?

**Comment by swarajpure** [comment]
> 
> 
> Can you add the Swagger docs for this route?

Added, Please check.

File: routes/members.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const router = express.Router()
+const membersController = require('../controllers/membersController')
+
+router.get('/', membersController.getMembers)
+
+/**
+ * @swagger
```
Comment (ankurnarkhede) [inline routes/members.js:8]: Add the annotations above the route initialization.

File: controllers/membersController.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  try {
+    const allMembers = await memberQuery.fetchMembers()
+
+    return res.json({
+      message: 'Members returned successfully!',
+      members: allMembers
+    })
+  } catch (error) {
+    logger.error(`Error while fetching all members: ${error}`)
+    return res.boom.serverUnavailable('Something went wrong. Please contact admin')
```
Comment (ankurnarkhede) [inline controllers/membersController.js:21]: Return 500, not 503.

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

File: routes/members.js (hunk 2 above)
Comment (swarajpure) [reply routes/members.js:6]: Done!

File: controllers/membersController.js (hunk 3 above)
Comment (swarajpure) [reply controllers/membersController.js]: Done. Changing in API-contracts too.

**Review by ankurnarkhede — APPROVED** [review]

**Pull request merged**